build.gradle

```diff
@@ -1,72 +1,79 @@
 buildscript {
-  repositories {
-    jcenter()
-  }
+    repositories {
+        jcenter()
+    }
 }
 
 plugins {
     id 'java'
     id 'application'
     id 'base'
     id 'nebula.lint' version '9.3.4'
+    id "com.github.hierynomus.license" version "0.15.0"
```

Comment:
Best to use single quotes in `build.gradle`/`groovy` unless you're doing string interpolation.